specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/gallery.json

```diff
@@ -0,0 +1,2708 @@
+{
+  "swagger": "2.0",
```

Comment:
The swagger file diff can be seen here: https://github.com/Azure/azure-rest-api-specs/pull/12860/files/2efd568efe6ba5b7367fa483d3e8f05a715118e2..4c8f0a24c28cd689782287af163643e021105574

The type change for the GalleryDiskImage Source property is a concern and should receive Breaking Change approval before merging.